docs/reference/hacl/dh/ec/curve25519.md

```diff
@@ -0,0 +1,92 @@
+# Curve25519
+
+## API Reference
+
+`````{tabs}
+````{group-tab} 51
+```C
+#include "Hacl_Curve25519_51.h"
+```
+````
+````{group-tab} 64
+```C
+#include "Hacl_Curve25519_64.h"
+```
+````
+````{group-tab} 64 (Slow)
```

Comment:
This is a 64-bit implementation that doesn't have the hardware requirements from above (and is pretty useless in my opinion) 🤷🏻

Reply:
What do you propose to do?

Reply:
I take that as a general remark. The versions are now briefly described.